src/ethereum/forks/osaka/vm/gas.py

```diff
@@ -84,6 +84,109 @@
 GAS_BLS_G2_MUL = Uint(22500)
 GAS_BLS_G2_MAP = Uint(23800)
 
+# EEST Specific Variables
+G_TX_DATA_ZERO = 4
+G_TX_DATA_NON_ZERO = 16
+
+# Transaction costs (from transactions.py)
+TX_BASE_COST = Uint(21_000)
+TX_CREATE_COST = Uint(32_000)
+TX_ACCESS_LIST_ADDRESS_COST = Uint(2_400)
+TX_ACCESS_LIST_STORAGE_KEY_COST = Uint(1_900)
+STANDARD_CALLDATA_TOKEN_COST = Uint(4)
+FLOOR_CALLDATA_COST = Uint(10)
```

Comment:
IIUC, these constants are _not_ used by anything under the `ethereum.forks.osaka.vm` package, right? It seems odd that they live in here if they aren't related to the virtual machine itself.